proto/alarm/v1/snapshot.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package alarmsnapshot.v1;
+
+import "aclk-schemas/proto/alarm/v1/stream.proto";
+
+option go_package = "alarm/snapshot/v1;alarmsnapshot";
```

Comment:
Also a quick question. Since you have everything under the same alarm/v1 dir why are we splitting each proto in a different package. It will be harder to import proto definitions accross packages.

What if you change all packages to be 
```
package alarm.v1;
option go_package = "alarm/v1;alarm";
```

Then you don't need to prefix the AlarmLogEntry with `alarmstream.v1.`

Reply:
Yes, indeed saw that... charts are following what you're suggesting, but not alarms... I didn't change the package to a generic `alarm.v1` because I didn't want to cause any issues on the already imported packages to the agent and the cloud... We'll see with Timo if that could be a problem...

Reply:
If there is a problem let's add the snapshot to the stream proto so we don't do any weird proto imports.

Reply:
I'd prefer to leave them in separate files... Just to keep them tidy and keep the same logic of charts... So basically change the `package` in all alarms to `alarms.v1`... Will do that...